pkgs/by-name/wl/wl-gammarelay-rs/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "wl-gammarelay-rs";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "maxverevkin";
+    repo = "wl-gammarelay-rs";
+    rev = "v${version}";
+    hash = "sha256-md6e9nRCs6TZarwFD3/GQEoJSIhtdq++rIZTP7Vl0wQ=";
+  };
+
+  cargoHash = "sha256-TDP5RC7B7/ldpK22WsmXd6fSl2rHtvG0hP9NYzoEVYo=";
+
+  nativeBuildInput = [stdenv.cc];
+
+  meta = {
+    description = "A simple program that provides DBus interface to control display temperature and brightness under wayland without flickering.";
+    homepage = "https://github.com/MaxVerevkin/wl-gammarelay-rs";
+    license = lib.licenses.gpl3Plus;
+    mainProgram = "wl-gammarelay-rs";
+    maintainers = with lib.maintainers; [quantenzitrone];
```

Comment:
Style nit.
```suggestion
    maintainers = with lib.maintainers; [ quantenzitrone ];
```

Reply:
nah

Reply:
The RFC you linked above (currently) says:

> - Brackets and braces are generally written with a space on the inside, like `[ `, ` ]`, `{ ` and ` }`.

So applying this suggestion would be consistent with applying the RFC's suggested formatting earlier.

<sub>But from personal experience, being overly pedantic about this not-yet-standardised formatting stuff is incredibly frustrating to deal with, so this isn't a blocking issue for me.</sub>

Reply:
well, the draft implementation doesn't fully adhere to the RFC
piegamesde implemented that change during an ongoing discussion https://github.com/NixOS/rfcs/pull/166#discussion_r1409883742